src/components/GameView/ReauthenticateDialog.vue

```diff
@@ -1,52 +1,56 @@
 <template>
-  <v-dialog v-model="show" persistent max-width="650">
-    <v-card id="reauthenticate-dialog">
-      <v-card-title>Reconnect to Game</v-card-title>
-      <form @submit.prevent="login">
-        <v-card-text>
-          <p>You have disconnected due to inactivity. Log in again to resume your session</p>
-          <!-- Login Form -->
-          <v-text-field
-            v-model="username"
-            variant="outlined"
-            :dense="$vuetify.display.mdAndDown"
-            label="Username"
-            data-cy="username"
-          />
-          <v-text-field
-            v-model="password"
-            variant="outlined"
-            label="Password"
-            :dense="$vuetify.display.mdAndDown"
-            type="password"
-            data-cy="password"
-          />
-        </v-card-text>
-        <v-card-actions class="d-flex justify-end">
-          <v-btn variant="text" color="primary" @click="leaveGame">
-            Leave Game
-          </v-btn>
-          <v-btn color="primary" variant="flat" data-cy="login" type="submit">
-            Log In
-          </v-btn>
-        </v-card-actions>
-      </form>
-    </v-card>
+  <div>
+    <base-dialog v-model="show" id="reauthenticate-dialog" title="Reconnect to Game">
+
+      <template #body>
+        <p>You have disconnected due to inactivity. Log in again to resume your session</p>
+        <!-- Login Form -->
+        <v-text-field
+          v-model="username"
+          variant="outlined"
+          :dense="$vuetify.display.mdAndDown"
+          label="Username"
+          data-cy="username"
+          @keydown.enter="login"
```

Comment:
This seems weird, the text fields should be submitting on enter anyway based on a form action. Not the end of the world though.

Reply:
This seems like a good compromise. I'll give it a shot

Reply:
Okay I am up for this. One gotcha is that we still need `@keydown.enter` handlers on the text fields because the form won't submit-on-enter when the form doesn't have a submit button. I considered adding a hidden submit button, but I think this is more clear

Reply:
Now it's this

```
        <form ref="form" @submit.prevent="login">
          <v-text-field
            v-model="username"
            variant="outlined"
            :dense="$vuetify.display.mdAndDown"
            label="Username"
            :loading="isLoggingIn"
            data-cy="username"
            @keydown.enter="submit"
          />
          <v-text-field
            v-model="password"
            variant="outlined"
            label="Password"
            :dense="$vuetify.display.mdAndDown"
            type="password"
            :loading="isLoggingIn"
            data-cy="password"
            @keydown.enter="submit"
          />
        </form>
      </template>

      <template #actions>
        <v-btn variant="text" color="primary" @click="leaveGame">
          Leave Game
        </v-btn>
        <v-btn
          color="primary"
          variant="flat"
          data-cy="login"
          :loading="isLoggingIn"
          @click="submit"
        >
          Log In
        </v-btn>
      </template>
```